README.md

```diff
@@ -1 +1 @@
-# kevinyou.github.io
\ No newline at end of file
+https://lichess.org/x9XK4Mkp
```

Comment:
I get a 404 when trying to access this:
```bash
$ curl -i https://lichess.org/x9XK4Mkp
HTTP/1.1 404 Not Found
Server: nginx
Date: Thu, 03 Nov 2022 04:32:27 GMT
```
Here is my understanding of the issue:
1. I looked up `ChessPursuit` in the lichess server source code and found [this 404 view](https://github.com/lichess-org/lila/blob/master/app/views/base/notFound.scala).
2. This view is rendered when the web app encounters a 404 from the API. (Citation needed)
3. The web app makes a call to the `GET /game/${}` API endpoint (Citation needed)
4. This API will return a 404 (when data for a user or game is not found](https://github.com/lichess-org/lila/blob/master/app/controllers/Api.scala#L469-L480) from the [database driver](https://github.com/lichess-org/lila/blob/master/modules/api/src/main/GameApi.scala#L82-L87).
5. The database driver uses [a Scala MongoDB driver](https://github.com/lichess-org/lila/blob/master/modules/game/src/main/GameRepo.scala#L26) known as [ReactiveMongo](http://reactivemongo.org/)
6. Game data is stored in a MongoDB instance, the "main db" in this [architecture diagram](https://raw.githubusercontent.com/lichess-org/lila/master/public/images/architecture.png).

Therefore, the issue is that the game no longer exists in the lichess main db.

@jhleung Please update this with a working game.

Reply:
thanks for checking man. i'll push up a change. i'll replace it with a generic expected string